examples/fiverthirtyeight-example/styles/globals.css

```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
```

Comment:
I noticed that the tables are looking kinda weird. I believe you are missing the PortalJS styles import